packages/manager/apps/dedicated/client/app/css/source.less

```diff
@@ -80,3 +80,16 @@ oui-message,
 h3.font-lite {
   font-weight: 300;
 }
+
+th .tooltip-container {
+  position: relative;
+  display: inline-block;
+  font-weight: 400;
+}
+
+td,
+th {
+  .oui-tooltip {
+    white-space: pre-wrap;
+  }
+}
```

Comment:
Could those styles be handled another way ? Boostrap utilities for instance ?

Reply:
I dont think there is a bootstrap class for `white-space`. Besides this should go away when `table` is migrated to `oui-datagrid` here